waspc/cli/Command/Call.hs

```diff
@@ -10,4 +10,7 @@ data Call
   | Version
   | Telemetry
   | Deps
+  | PrintBashCompletionInstruction
+  | GenerateBashCompletionScript
+  | Commands [String]
```

Comment:
Let's rename `Commands` to `BashCompletionListCommands`, to be synced with rest of the code?